gpflow/kernels/misc.py

```diff
@@ -98,51 +98,66 @@ def K_diag(self, X):
 
 
 class Coregion(Kernel):
-    def __init__(self, output_dim, rank, active_dims=None):
-        """
-        A Coregionalization kernel. The inputs to this kernel are _integers_
-        (we cast them from floats as needed) which usually specify the
-        *outputs* of a Coregionalization model.
+    """
+    A Coregionalization kernel. The inputs to this kernel are _integers_ (we
+    cast them from floats as needed) which usually specify the *outputs* of a
+    Coregionalization model.
 
-        The parameters of this kernel, W, kappa, specify a positive-definite
-        matrix B.
+    The kernel function is an indexing of a positive-definite matrix:
 
-          B = W W^T + diag(kappa) .
+      K(x, y) = B[x, y] .
 
-        The kernel function is then an indexing of this matrix, so
+    To ensure that B is positive-definite, it is specified by the two
+    parameters of this kernel, W and kappa:
 
-          K(x, y) = B[x, y] .
+      B = W Wᵀ + diag(kappa) .
 
-        We refer to the size of B as "num_outputs x num_outputs", since this is
-        the number of outputs in a coregionalization model. We refer to the
-        number of columns on W as 'rank': it is the number of degrees of
-        correlation between the outputs.
+    We refer to the size of B as "output_dim x output_dim", since this is the
+    number of outputs in a coregionalization model. We refer to the number of
+    columns on W as 'rank': it is the number of degrees of correlation between
+    the outputs.
 
-        NB. There is a symmetry between the elements of W, which creates a
-        local minimum at W=0. To avoid this, it's recommended to initialize the
-        optimization (or MCMC chain) using a random W.
+    NB. There is a symmetry between the elements of W, which creates a local
+    minimum at W=0. To avoid this, it is recommended to initialize the
+    optimization (or MCMC chain) using a random W.
+    """
+
+    def __init__(self, output_dim: int, rank: int, **kwargs):
+        """
+        :param output_dim: number of outputs expected (0 <= X < output_dim)
+        :param rank: number of degrees of correlation between outputs
         """
 
         # assert input_dim == 1, "Coregion kernel in 1D only"
-        super().__init__(active_dims)
+        super().__init__(**kwargs)
 
         self.output_dim = output_dim
         self.rank = rank
-        W = np.zeros((self.output_dim, self.rank))
+        W = np.ones((self.output_dim, self.rank))
```

Comment:
as I said before, I don't know the implications of randn vs ones vs sth else - you'll need to ask sbd else about this

Reply:
double-checked with @jameshensman